src/Fixer/Phpdoc/PhpdocVarWithoutNameFixer.php

```diff
@@ -68,21 +68,30 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
-            $doc = new DocBlock($token->getContent());
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+            $nextToken = isset($tokens[$nextIndex]) ? $tokens[$nextIndex] : null;
 
-            // don't process single line docblocks
-            if (1 === count($doc->getLines())) {
-                continue;
+            // For people writing static public $foo instead of public static $foo
+            if ($nextToken && $nextToken->isGivenKind([T_STATIC])) {
```

Comment:
Better to be strict and check `$nextToken !== null` or `$nextToken instanceof Token`